### DISCUSSION TITLE
[Feature]May I grant permission? I used some code in my script

### DISCUSSION THREAD

**Discussion by xiaobaitulele** [redphx/better-xcloud#62] [General]
How to get some license？No contact information .

**Comment by xiaobaitulele** [comment]
https://greasyfork.org/zh-CN/scripts/455741

**Comment by redphx** [comment]
Just follow the MIT license's requirements. And you must let users know that you use code from this project and have a link back to it.

**Reply by xiaobaitulele** [reply]
![image](https://github.com/redphx/better-xcloud/assets/141475916/474408a5-244d-4714-a916-bdff70adddd4)

ok,The description has been updated
